client.py

```diff
@@ -0,0 +1,57 @@
+import sys
+import json
+import socket
+import time
+from common.variables import ACTION, PRESENCE, TIME, USER, ACCOUNT_NAME, \
+    RESPONSE, ERROR, DEFAULT_IP_ADDRESS, DEFAULT_PORT
+from common.utils import get_message, send_message
+
+
+def create_presence(account_name='Guest'):
+    # {'action': 'presence', 'time': 1573760672.167031, 'user': {'account_name': 'Guest'}}
+    out = {
+        ACTION: PRESENCE,
+        TIME: time.time(),
+        USER: {
+            ACCOUNT_NAME: account_name
+        }
+    }
+    return out
+
+
+def process_ans(message):
+    if RESPONSE in message:
+        if message[RESPONSE] == 200:
+            return '200 : OK'
+        return f'400 : {message[ERROR]}'
+    raise ValueError
+
+
+def main():
+    # client.py 192.168.1.2 8079
+    try:
+        server_address = sys.argv[1]
+        server_port = int(sys.argv[2])
+        if server_port < 1024 or server_port > 65535:
+            raise ValueError
+    except IndexError:
+        server_address = DEFAULT_IP_ADDRESS
+        server_port = DEFAULT_PORT
+    except ValueError:
+        print('В качестве порта может быть указано только '
+              'число в диапазоне от 1024 до 65535.')
+        sys.exit(1)
+
+    transport = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    transport.connect((server_address, server_port))
+    message_to_server = create_presence()

```

Comment:
Wrong port is not handled:
```
Traceback (most recent call last):
  File "C:/Projects/Python/Client-server-applications-in-python/client.py", line 57, in <module>
    main()
  File "C:/Projects/Python/Client-server-applications-in-python/client.py", line 46, in main
    transport.connect((server_address, server_port))
ConnectionRefusedError: [WinError 10061] No connection could be made because the target machine actively refused it

Reply:
client.py 127.0.0.1 7771